open-metadata-implementation/governance-servers/virtualization-services/virtualization-services-api/src/main/java/org/odpi/openmetadata/governanceservers/virtualizationservices/event/VirtualizerTopicListener.java

```diff
@@ -59,10 +59,12 @@ public void processEvent (String event){
         ObjectMapper objectMapper = new ObjectMapper();
         try{
             TableContextEvent eventObject = objectMapper.readValue(event, TableContextEvent.class);
```

Comment:
Please note that on the topic there are multiple events published, all extending InformationViewHeader. Virtualizer should handle only TableContextEvent, but this code throws exception for all other events. An example of such an event is SemanticAssignment. These events should be discarded without the stacktrace being printed. A suggestion is to use a condition similar to _new ObjectMapper().readValue(json, InformationViewHeader.class) instanceof TableContextEvent_, but it is up to you to use any condition and define the code logic